system-addon/content-src/components/Search/Search.jsx

```diff
@@ -0,0 +1,34 @@
+"use strict";
+const React = require("react");
+const {connect} = require("react-redux");
+const {actionTypes, actionCreators} = require("common/Actions.jsm");
+
+const Search = React.createClass({
+  getInitialState() {
+    return {
+      searchString: ''
```

Comment:
nit: Use double quotes as used everywhere else (thought there was an eslint rule for this??)